### DISCUSSION TITLE
Add ability to pin comments in really long issue threads

### DISCUSSION THREAD

**Discussion by karrtikr** [community/community#27522] [Projects and Issues]
There are some issues which have been solved for a long time, but the threads are that people have a difficult time finding the workaround until an issue is solved. It would be nice for collabs to have a way to pin a comment which is the "tried-and-true" way to solve a certain issue.

**Comment by gnowland** [comment]
This is a duplicate of:

- https://github.com/orgs/community/discussions/18523
- https://github.com/orgs/community/discussions/47912

as well as (from the now archived `isaacs/github` repository:

- https://github.com/isaacs/github/issues/1008 (which contains some examples and references to related feature requests)
- https://github.com/isaacs/github/issues/1555

**Reply by cwway** [reply]
Added mock-up [here](https://github.com/orgs/community/discussions/47912#discussioncomment-14587531)
